packages/ifpack2/src/Ifpack2_BlockRelaxation_def.hpp

```diff
@@ -141,6 +141,9 @@ setParameters (const Teuchos::ParameterList& List)
     if (relaxType == "Jacobi") {
       PrecType_ = Ifpack2::Details::JACOBI;
     }
+    else if (relaxType == "MTJacobi") {
+      PrecType_ = Ifpack2::Details::MTJACOBI;
+    }
```

Comment:
@mhoemmen, the line smoother does not use graph coloring. It's threaded (on CPU, KNL) over lines. So "Multithreaded" might be the way to go. "Multicore" is not, IMO.